templates/index.html

```diff
@@ -1,27 +1,968 @@
+                >
+                  Add
+                </button>
+              </footer>
+            </div>
+            <div class="mdc-dialog__backdrop"></div>
+          </aside>
+        </div>
+      </main>
+    </div>
+
+    <div class="crypto-menu-switches">
+      <div class="crypto-menu-switches--handle">
+        <i class="material-icons">settings</i>
+      </div>
+      <!-- RTL switch -->
+      <div class="switch-wrapper mdc-theme--on-primary">
+        <div class="mdc-switch crypto-rtl-switch-selector">
+          <div class="mdc-switch__track"></div>
+          <div class="mdc-switch__thumb-underlay">
+            <div class="mdc-switch__thumb">
+              <input
+                type="checkbox"
+                id="rtl-switch"
+                class="mdc-switch__native-control"
+                role="switch"
+                aria-checked="false"
+              />
+            </div>
+          </div>
+        </div>
+        <label for="rtl-switch">RTL</label>
+      </div>
+      <!-- theme switch -->
+      <div class="switch-wrapper mdc-theme--on-primary">
+        <div class="mdc-switch crypto-theme-switch-selector">
+          <div class="mdc-switch__track"></div>
+          <div class="mdc-switch__thumb-underlay">
+            <div class="mdc-switch__thumb">
+              <input
+                type="checkbox"
+                id="crypto-theme-switch"
+                class="mdc-switch__native-control"
+                role="switch"
+                aria-checked="false"
+              />
+            </div>
+          </div>
+        </div>
+        <label for="crypto-theme-switch">Switch theme</label>
+      </div>
+    </div>
+
+    <!-- initialise material components js -->
+    <script src="https://code.jquery.com/jquery-1.9.1.min.js"></script>
+    <script src="{{ url_for('static', path='/assets/js/material-components-web.min.js') }}"></script>
+    <script>
+      mdc.autoInit();
+    </script>
+    <!-- initialise marquee js -->
+    <script src="{{ url_for('static', path='/assets/js/marquee3k.js') }}"></script>
+    <script>
+      Marquee3k.init();
+    </script>
+    <!-- layout js -->
+    <script
+      data-pace-options='{ "ajax": false, "restartOnRequestAfter": false }'
+      src="{{ url_for('static', path='/assets/js/pace.min.js') }}"
+    ></script>
+    <script src="{{ url_for('static', path='/assets/js/apexcharts.min.js') }}"></script>
+    <!-- switcher js -->
+    <script src="{{ url_for('static', path='/assets/js/partials/switcher.js') }}"></script>
+    <!-- page specific JS -->
+    <script
+      type="text/javascript"
+      loading="lazy"
+      src="https://www.gstatic.com/charts/loader.js"
+    ></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+
+    <script src="{{ url_for('static', path='/assets/js/pages/index.js') }}"></script>
+    <script src="{{ url_for('static', path='/assets/js/requests.js') }}"></script>
```

Comment:
Remove this for now because `/assets/js/requests.js` doesn't have any functionality right now.